x/signal/keeper_test.go

```diff
@@ -26,6 +23,9 @@ import (
 	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	"math"
+	"math/big"
+	"testing"
```

Comment:
would be nice to setup import ordering at some point for the project. It's a bit wack in many of the packages

Reply:
yeah it is, I'll create an issue for it, this was just my editor auto formatting.

Reply:
https://linear.app/binarybuilders/issue/CEL-39/format-lint-the-repo

Reply:
eventually a make lint-fix would solve those